test/LibraryManager.IntegrationTest/LibmanTestsUtility.cs

```diff
@@ -8,6 +9,164 @@ namespace Microsoft.Web.LibraryManager.IntegrationTest
 {
     public class LibmanTestsUtility
     {
+        private static HashSet<string> GetTopLevelDirectoriesAndFiles(string cwd, bool caseInsensitive)
+        {
+            HashSet<string> topLevelItems = caseInsensitive ? new HashSet<string>(StringComparer.OrdinalIgnoreCase) : new HashSet<string>();
+            var dir = new DirectoryInfo(cwd);
+
+            if(dir.Exists)
+            {
+                foreach(FileSystemInfo item in dir.EnumerateDirectories())
```

Comment:
Is this the same as Directory.EnumerateFileSystemEntries ?

Reply:
+1.  Also, shouldn't this be recursive?  You could just use `Directory.EnumerateFileSystemEntries(path, "*", SearchOption.AllDirectories)`

---
In reply to: [194396415](https://github.com/aspnet/LibraryManager/pull/113#discussion_r194396415) [](ancestors = 194396415)

Reply:
(That could replace this whole method)

---
In reply to: [194556460](https://github.com/aspnet/LibraryManager/pull/113#discussion_r194556460) [](ancestors = 194556460,194396415)

Reply:
During the test design discussion, we are going to verify only top level items. If it's not too much work, I think verifying the sub-folder items also make sense to me.

---
In reply to: [194556684](https://github.com/aspnet/LibraryManager/pull/113#discussion_r194556684) [](ancestors = 194556684,194556460,194396415)